torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2491,6 +2491,25 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
                         kwargs={'alpha': alpha_val, 'beta': beta_val},))
     return sample_inputs
 
+def sample_inputs_sparse_sampled_addmm(op_info, device, dtype, requires_grad, **kwargs):
+    alpha = 2 + 3j if dtype.is_complex else 0.6
+    beta = 1 + 2j if dtype.is_complex else 0.2
+
+    def generator():
+        # sparse.sampled_addmm performs: alpha * (A @ B) * sparse_ones_like(C) + beta * C
+        for index_dtype in (torch.int32, torch.int64):
+            for m, n, k in itertools.product([0, 5], repeat=3):
+                yield SampleInput(
+                        make_tensor((m, n), device=device, dtype=dtype).to_sparse_csr().requires_grad_(requires_grad),
```

Comment:
you could make sure that there are some zeros in here by e.g. calling relu() on the result. it might make for a more powerful test, since I think make_tensor will just give you randn. We can do this in a follow-up.

Reply:
Sure, I will change that.